src/protocols/BaseProtocolAdapter.ts

```diff
@@ -1,9 +1,22 @@
-import type { IHdWalletAccount } from '@/types';
+import BigNumber from 'bignumber.js';
+import type {
+  ICoin,
+  IHdWalletAccount,
+} from '@/types';
 
 /**
  *  Represents common attributes and behavior of a protocol
  */
 export abstract class BaseProtocolAdapter {
+  abstract getCoingeckoCoinId(): string;
```

Comment:
```suggestion
  abstract getCoinGeckoCoinId(): string;
```
Coingecko => CoinGecko